src/libraries/System.Formats.Tar/src/System/Formats/Tar/UstarTarEntry.cs

```diff
@@ -36,6 +36,11 @@ public UstarTarEntry(TarEntryType entryType, string entryName)
         /// <summary>
         /// Initializes a new <see cref="UstarTarEntry"/> instance by converting the specified <paramref name="other"/> entry into the Ustar format.
         /// </summary>
+        /// <param name="other">The <see cref="TarEntry"/> instance to convert to the Ustar format.</param>
+        /// <exception cref="InvalidOperationException"><para><paramref name="other"/> is a <see cref="PaxGlobalExtendedAttributesTarEntry"/> instance.</para>
+        /// <para>-or-</para>
+        /// <para>The entry type of <paramref name="other"/> is not supported in Ustar.</para></exception>
```

Comment:
```suggestion
        /// <para>The entry type of <paramref name="other"/> is not supported in the Ustar format.</para></exception>
```
BTW, when we're not referring to our own enum, should it be cased as "UStar" ?